spec/lib/open_food_network/enterprise_injection_data_spec.rb

```diff
@@ -5,13 +5,6 @@ module OpenFoodNetwork
     describe "relatives" do
       let!(:enterprise) { create(:distributor_enterprise) }
       let!(:producer) { create(:supplier_enterprise) }
-      let!(:producer_inactive) { create(:supplier_enterprise, confirmed_at: nil) }
-      let!(:er_p)  { create(:enterprise_relationship, parent: producer, child: enterprise) }
-      let!(:er_pi) { create(:enterprise_relationship, parent: producer_inactive, child: enterprise) }
-
-      it "only loads activated relatives" do
-        subject.relatives[enterprise.id][:producers].should_not include producer_inactive.id
-      end
```

Comment:
I think we should be able to use `sells: 'unconfirmed'` instead of `confirmed_at: nil` here too...